rflx/const.py

```diff
@@ -2,3 +2,8 @@
 
 BUILTINS_PACKAGE = ID("__BUILTINS__")
 INTERNAL_PACKAGE = ID("__INTERNAL__")
+
+# ISSUE: Componolit/RecordFlux#1077
+# size of integers is limited to 63bits
+
+MAX_FIELD_SIZE = 63
```

Comment:
Wouldn't fit `MAX_SCALAR_SIZE` better? The current restriction does not only limit message fields to 63 bit, but all scalar types in all scenarios (e.g., also scalar types which are used in sessions).

Reply:
I agree, done